src/stavetempo.ts

```diff
@@ -75,21 +76,22 @@ export class StaveTempo extends StaveModifier {
     const y = stave.getYForTopText(1) + this.shift_y;
 
     ctx.save();
+    const textFormatter = TextFormatter.create(this.textFont);
 
     if (name) {
       ctx.setFont(this.textFont);
       ctx.fillText(name, x, y);
-      x += ctx.measureText(name).width;
+      x += textFormatter.getWidthForTextInPx(name);
```

Comment:
I have made the following change to check what is the difference:
```
      ctx.fillText(name, x, y);
      ctx.fillRect(x, y + 8, ctx.measureText(name).width, 2);
      ctx.fillRect(x, y + 13, textFormatter.getWidthForTextInPx(name), 2);
      x += textFormatter.getWidthForTextInPx(name);
```
and I get

![image](https://user-images.githubusercontent.com/22865285/196111145-063a8ac1-cd46-4989-8ebd-bf1a50da150a.png)

@AaronDavidNewman I would say that `meassureText` provides the right `width`.